lib/internal/bootstrap/pre_execution.js

```diff
@@ -158,6 +165,36 @@ function setupSignalHandlers() {
   process.on('removeListener', stopListeningIfSignal);
 }
 
+
+function setupStacktracePrinterOnSigint() {
+  // Use tty_wrap directly to produce minimal require footprint.
+  const { isTTY } = internalBinding('tty_wrap');
+  if (!isTTY(/** stderr */2)) {
```

Comment:
```suggestion
  if (!isTTY(process.stderr.fd)) {
```

Reply:
This code should not have to instantiate process.stderr.

Reply:
to clarify, instantiating stdio has side effects as it starts buffering.